site/src/content/blog/force-push-without-overwriting-work.md

```
@@ -0,0 +1,266 @@
+where you last saw it — a question about your bookkeeping, and any
+fetch answers it for you. `--force-if-includes` asks whether the
+remote's tip is part of the history you're pushing — a question about
+the work, and only integrating that commit answers it. A fetch can't
+fake the answer. Neither can a cherry-picked copy of the commit:
+the check wants the remote tip itself reachable in your branch,
+not a lookalike.
+
+The flag is easy to make permanent:
+
+```sh
+$ git config push.useForceIfIncludes true
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)